CHANGES.rst

```diff
@@ -178,9 +178,10 @@ Version 1.0.2
 
 Unreleased
 
--   Add new "edg" identifier for Edge in UserAgentPreparser.
+-   Add new "edg" identifier for Edge in ``UserAgentParser``.
     :issue:`1797`
 -   Upgrade the debugger to jQuery 3.5.1. :issue:`1802`
+-   Updated ``UserAgentParser`` to handle more cases. :issue:`1971`
```

Comment:
Looks like you added this to an old section while fixing the code formatting above. This line should be in the 2.0.0 section.